### PR TITLE
Add a list of 'users of this chart'

### PR DESCRIPTION
**Pull request by yuvipanda**
Useful for showing wide usage

### CODE REVIEW

**Comment by yuvipanda** [comment]
We might maybe want to add a one-line description. Not sure.

/cc @choldgraf @willingc if you know other people who should be on this list (plus thoughts on how to structure this).

**Comment by choldgraf** [comment]
I think it's a good idea...we could turn this into some kind of gallery if this list ever gets long enough w/ public-facing deployments

**Comment by yuvipanda** [comment]
+1. Do we also want to count the workshops and what not that people use this for?

**Comment by choldgraf** [comment]
Why not? We can have a workshops section? Or maybe this would go into some kind of place like #278 specifies?

Also we should provide some way for others to tell us about public jupyterhub deployments. Maybe ask people to make a PR if they want to mention their deployment?

**Comment by yuvipanda** [comment]
Cool, I'm all for having a workshop section :) wanna add a few to this list? Between you, @arokem and @aculich that's a few. Let's only include workshops about various subjects that was conducted using z2jh, rather than z2jh workshops themselves...

**Comment by choldgraf** [comment]
yeah I agree, off the top of my head:

* Neurohackweek
* Data on the mind
* The South Data Hub talk


but there are definitely others.

**Comment by willingc** [comment]
I recommend something similar to: http://jupyterhub.readthedocs.io/en/latest/gallery-jhub-deployments.html

File: doc/source/users-list.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Zero to JupyterHub users
```
Comment (willingc) [inline doc/source/users-list.md]: I would change the title to Gallery to give more flexibility over time with content

**Comment by choldgraf** [comment]
+1 to merge after we rename to @willingc 's "Gallery" suggestion. That way we don't have to create a new thing whenever we get around to creating images etc for the deployments.

**Comment by choldgraf** [comment]
I went ahead and made @willingc 's suggestion, merging now!

**Comment by choldgraf** [comment]
Is travis being funky right now? I restarted it once but have run into two time-out errors...

**Comment by yuvipanda** [comment]
@choldgraf travis was being funky, I filed https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/434 to investigate and a more permanent fix in https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/435

**Comment by choldgraf** [comment]
Cool! Merged this dangling PR :-)

**Pull request merged**